core/controller/src/main/scala/whisk/core/loadBalancer/ShardingContainerPoolBalancer.scala

```diff
@@ -74,6 +73,9 @@ class ShardingContainerPoolBalancer(config: WhiskConfig, controllerInstance: Ins
   private val activationsPerNamespace = TrieMap[UUID, LongAdder]()
   private val totalActivations = new LongAdder()
 
+  var localOverflowActivationCount: Int = 0
```

Comment:
If the topic receives messages from multiple controllers, this count won't reflect whether there are messages left in the topic, I think? For example, if another controller consumes ALL the overflow messages, will this controller be left indefinitely with localOverflowActivationCount > 0?